src/Readability.php

```diff
@@ -1076,11 +1079,6 @@ protected function grabArticle(?\DOMElement $page = null)
                 }
             }
 
-            $candidates = $xpath->query('.//*[not(self::body) and (@class or @id or @style) and ((number(@readability) < 40) or not(@readability))]', $page->documentElement);
-
-            for ($c = $candidates->length - 1; $c >= 0; --$c) {
-                $node = $candidates->item($c);
-            }
```

Comment:
Why did you remove that part?

Reply:
https://github.com/jtojnar/php-readability/commit/b580cf216d9001f82c866bb9a6c8bcad1cc862d8 moved the code that used the `$node` variable so now it is not used as far as I can tell.